boards/arm/nucleo_h743zi/doc/index.rst

```diff
@@ -0,0 +1,204 @@
+.. _nucleo_h743zi_board:
+
+ST Nucleo H743ZI
+################
+
+Overview
+********
+
+The STM32 Nucleo-144 boards offer combinations of performance and power that
+provide an affordable and flexible way for users to build prototypes and try
+out new concepts. For compatible boards, the SMPS (Switching Mode Power Supply)
```

Comment:
While not really critical, the industry standard term is "switched-mode power supply" (or alternatively "switch mode power supply).

When saying "switching," you don't really put "mode" there (so, "switching power supply" is acceptable but "switching mode power supply" is not).

Don't ask me why ...

Reply:
Fixed, thanks for the clarification